dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpRequestMessageImpl.java

```diff
@@ -338,7 +338,7 @@ protected void setPseudoHeaders(HashMap<String, String> pseudoHeaders) throws Ex
             this.setScheme(pseudoHeaders.get(HpackConstants.SCHEME));
         }
         if (pseudoHeaders.containsKey(HpackConstants.AUTHORITY)) {
-            parseH2Authority(pseudoHeaders.get(HpackConstants.AUTHORITY).getBytes());
+            parseH2Authority(pseudoHeaders.get(HpackConstants.AUTHORITY).getBytes(GenericUtils.CHARSET_FOR_GETBYTES));
```

Comment:
Can this just be changed to:
```suggestion
            parseH2Authority(GenericUtils.getBytes(pseudoHeaders.get(HpackConstants.AUTHORITY));
```
and then just undo the changes in GenericUtils?

Reply:
Unsure

Reply:
Based off of what I am reading like other header values, it is just characters 0 to 127 so I made the change as I put above.